website/docs/components/app-side-nav/partials/code/how-to-use.md

```diff
@@ -206,15 +206,13 @@ The App Side Nav component **automatically**:
 
 Any other content in the App Side Nav needs to be **explicitly handled** by the consumers (in this way they have full control of the content they add, and they can customize the transition as they want/need).
 
-One possible way to do it is to use the **`hds-app-side-nav-hide-when-minimized` class**. This is a special class that can be applied to a DOM element so that it **automatically** fades in/out when the App Side Nav changes its “minimization” state.
+One possible way to do it is to use the **`hds-app-side-nav--hide-when-minimized` class**. This is a special class that can be applied to a DOM element so that it **automatically** fades in/out when the App Side Nav changes its “minimization” state.
```

Comment:
But this class is always present now, isn't it?

Reply:
Yes... Ok I have looked into it more and I believe we can safely simplify this logic. I will push up a refactor shortly.

Reply:
Just to close out the comment. We ended up refactoring the component and removing this class entirely. The whole content of the AppSideNav body is animated in/out so consumers dont need this escape hatch.